multi-node-k8s.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+
+# TODO: disable debugging
+set -x
+
+# get latest stable version, strip the leading "v"
+K8S_VERSION="$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)"
+
+# configure cluster parameters
+WORKERS=2
+WORKER_MEM=5
+NO_OF_DISKS=2
```

Comment:
maybe just `DISKS` as you also have `WORKERS`, or `DISKS_PER_WORKER` but that gets a little long

Reply:
Thanks. I'll make the changes.